src/meta/expr.lean

```diff
@@ -247,6 +247,14 @@ meta def fold_mvar {α} : level → (name → α → α) → α → α
 | (max a b) f := fold_mvar a f ∘ fold_mvar b f
 | (imax a b) f := fold_mvar a f ∘ fold_mvar b f
 
+/-- `u.params` is the set of parameters occuring in `u`. -/
```

Comment:
An example would have helped me read this
```suggestion
/-- `u.params` is the set of parameters occuring in `u`.
For instance `params` on `max u (max v w)` gives `{u, v, w}`. -/
```
(Is that reading true?)

Reply:
Yes, that is correct.